### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Brian-Doucet**
Fixed changes from previous pull request, based on feedback.

### CODE REVIEW

File: covid19/utils.py [hunk 1]
```diff
@@ -11,21 +10,19 @@
...
     Raises:
-        Exception: If no values in column specified then file is considered empty
+        Exception: If no values in file passed then file is considered empty
 
     Returns:
         list -- A list with all values from the column given
     """
-    file_to_df = pd.read_csv(file, usecols=[column])
-    list_of_values = file_to_df[column].tolist()
+    list_of_values = pd.read_csv(file, squeeze=True).tolist()
```
Comment (ilyayudkovich) [inline covid19/utils.py:26]: what does squeeze do?

File: covid19/utils.py [hunk 2]
```diff
@@ -95,14 +90,14 @@ def get_formatted_datetime(date_to_format: datetime) -> str:
...
-def is_valid_end_date(end_date: datetime) -> bool:
+def is_valid_end_date(end_date):
     """Check that date is prior to (or inclusive of) yesterday
 
     Arguments:
         end_date {datetime} -- The date to check
 
     Returns:
-        bool -- True if date entered is not greater than yesterday
+        None -- True if date entered is not greater than yesterday
```
Comment (ilyayudkovich) [inline covid19/utils.py]: so this still return a boolean value, not a None

File: covid19/utils.py (hunk 1 above)
Comment (Brian-Doucet) [reply covid19/utils.py:26]: It converts a DataFrame into a series (single column). The last version of this function I was creating the series with the usecols=[column] argument and then converting into a list on the next line. This method removes the extra argument `column` and condenses into a single line.

File: covid19/utils.py [hunk 3]
```diff
@@ -95,14 +90,14 @@ def get_formatted_datetime(date_to_format: datetime) -> str:
     return date_to_format.strftime(DATETIME_FORMAT)
 
 
-def is_valid_end_date(end_date: datetime) -> bool:
```
Comment (ilyayudkovich) [inline covid19/utils.py:98]: why'd you remove the static typing?

**Review by ilyayudkovich — COMMENTED** [review]
comment

**Pull request merged**